### PR TITLE
Fangamer-LLC.Deltarune version 0.6.6.6

### PR DESCRIPTION
**Pull request by Danmyrer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20478)

### CODE REVIEW

File: manifests/f/Fangamer-LLC/Deltarune/0.6.6.6/Fangamer-LLC.Deltarune.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Fangamer-LLC.Deltarune
+PackageVersion: 0.6.6.6
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://thisisatestofthevideogamedeliverysystem.sfo2.cdn.digitaloceanspaces.com/SURVEY_PROGRAM_WINDOWS_ENGLISH.exe?timestamp=1626009843&signature=2877254936
```
Comment (ItzLevvie) [inline manifests/f/Fangamer-LLC/Deltarune/0.6.6.6/Fangamer-LLC.Deltarune.installer.yaml:9]:
```suggestion
  InstallerUrl: https://thisisatestofthevideogamedeliverysystem.sfo2.cdn.digitaloceanspaces.com/SURVEY_PROGRAM_WINDOWS_ENGLISH.exe
  Scope: machine
```

File: manifests/f/Fangamer-LLC/Deltarune/0.6.6.6/Fangamer-LLC.Deltarune.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Fangamer-LLC.Deltarune
```
Comment (ItzLevvie) [inline manifests/f/Fangamer-LLC/Deltarune/0.6.6.6/Fangamer-LLC.Deltarune.yaml:4]:
```suggestion
PackageIdentifier: Fangamer.Deltarune
```

File: manifests/f/Fangamer-LLC/Deltarune/0.6.6.6/Fangamer-LLC.Deltarune.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Fangamer-LLC.Deltarune
```
Comment (ItzLevvie) [inline manifests/f/Fangamer-LLC/Deltarune/0.6.6.6/Fangamer-LLC.Deltarune.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Fangamer.Deltarune
```

File: manifests/f/Fangamer-LLC/Deltarune/0.6.6.6/Fangamer-LLC.Deltarune.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Fangamer-LLC.Deltarune
```
Comment (ItzLevvie) [inline manifests/f/Fangamer-LLC/Deltarune/0.6.6.6/Fangamer-LLC.Deltarune.installer.yaml:4]:
```suggestion
PackageIdentifier: Fangamer.Deltarune
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Danmyrer** [comment]
TODO:
- [ ] Change Fangamer-LLC to Fangamer consistently

**Comment by ItzLevvie** [comment]
Folder name too.

**Pull request closed without merging**